weather_sp/splitter_pipeline/file_splitters.py

```diff
@@ -90,89 +85,132 @@ def should_skip(self):
 
 class GribSplitter(FileSplitter):
 
+    def __init__(self, input_path: str, output_info: OutFileInfo,
+                 force_split: bool = False, logging_level: int = logging.INFO):
+        super().__init__(input_path, output_info,
+                         force_split, logging_level)
+        self.output_info.set_formatting_if_needed('_{typeOfLevel}_{shortName}')
+        self.split_dims = self._get_split_dims()
+
     def split_data(self) -> None:
+        if not self.split_dims:
+            raise ValueError('No splitting specified in template.')
         outputs = dict()
 
         if self.should_skip():
             metrics.Metrics.counter('file_splitters', 'skipped').inc()
-            self.logger.info('Skipping %s, file already split.', repr(self.input_path))
+            self.logger.info('Skipping %s, file already split.',
```

Comment:
Optional nit for the format string: it could be:
```
self.logger.info('Skipping {self.input_path!r}, file already split.')
```
Though... is there a performance reason why you'd want to keep it the same? I remember discussing this with @fredzyda, and I don't remember the conclusion of this conversation.

Reply:
I'm pretty sure that I read that for logging %s style is used over {} python style.